src/pages/Admin/Venue/DetailsForm.tsx

```diff
@@ -181,28 +182,32 @@ export const DetailsForm: React.FC<DetailsFormProps> = ({
             },
             user
           );
-
           //@debt Create separate function that updates the userStatuses separately by venue id.
           if (
             sovereignVenueId &&
             sovereignVenue &&
```

Comment:
Is `sovereignVenue` needed anywhere?
Should `fetchSovereignVenue()` be called before the `if`?

Reply:
still needed for the `DetailsFormLeft` render 
it's alright to call it after if since sovereign venue is to be updated inside it